spec/jobs/resolution_proofing_job_spec.rb

```diff
@@ -46,6 +46,32 @@
       )
     end
 
+    context 'ssn_is_unique attribute' do
+      context 'when the SSN is unique' do
+        it 'sets ssn_is_unique: true on the result' do
+          stub_vendor_requests
+          perform
+
+          result = document_capture_session.load_proofing_result[:result]
+          expect(result[:ssn_is_unique]).to be_truthy
+        end
+      end
+
+      context 'when the SSN is not unique' do
+        before do
+          allow_any_instance_of(Idv::DuplicateSsnFinder).to receive(:ssn_is_unique?).and_return false
+        end
+
+        it 'sets ssn_is_unique: false on the result' do
+          stub_vendor_requests
+          perform
+
+          result = document_capture_session.load_proofing_result[:result]
+          expect(result[:ssn_is_unique]).to be_falsey
```

Comment:
same as be-truthy. `nil` is falsey, so if there was a typo in the keyname, this spec would not catch the error
```suggestion
          expect(result[:ssn_is_unique]).to eq(false)
```